.infra/crons.ts

```diff
@@ -114,4 +114,8 @@ export const crons: Cron[] = [
     name: 'calculate-top-readers',
     schedule: '0 2 1 * *',
   },
+  {
+    name: 'clean-gifted-plus',
+    schedule: '0 1 * * *',
```

Comment:
```suggestion
    schedule: '45 2 * * *',
```

lets push it a bit so it does not overlap with other crons (just load wise)

its also good to not put it on even hour because usually other stuff happens in even hour